src/angularjs/src/app/places/compare/compare.controller.js

```diff
@@ -31,22 +31,23 @@
             });
 
             // first request score metadata
-            promises.unshift(ScoreMetadata.query().$promise);
+            promises.unshift(ScoreMetadata.query());
 
             // do not display any place until all places have been retrieved
             $q.all(promises).then(function(results) {
                 // first element is the metadata; rest are the places
-                var metadata = _.reduce(_.head(results), function (result, score) {
-                    result[score.name] = {
-                        label: score.label,
-                        category: score.category,
-                        description: score.description
-                    };
-                    return result;
-                }, {});
-                ctl.metadata = metadata;
+                ctl.metadata = _.head(results);
 
                 ctl.places = _.drop(results);
+                _.forEach(ctl.places, function (place) {
+                    place.scores = _.reduce(place.scores, function (result, score) {
+                        result[score.metric] = {
+                            score: score.score,
+                            score_normalized: score.score_normalized
+                        }
+                        return result;
+                    }, {});
+                })
```

Comment:
Two questions on this and its companion below:
1. It's converting `place.scores` to an object keyed by metric rather than a list of objects, right?  If I'm reading that right, then
```
place.scores = _.keyBy(place.scores, 'metric');
```
would accomplish the same thing.  (The same except that it would also leave the `metric` key in the object, which is a bit redundant but harmless.)
2. If this is how we want the data, couldn't we just do it in [`places.service.js`](https://github.com/azavea/pfb-network-connectivity/blob/develop/src/angularjs/src/app/components/places.service.js#L53)?  I believe these are the only two places where `Places.getPlace` is used.